x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/entity_store_data_client.ts

```diff
@@ -241,19 +241,19 @@ export class EntityStoreDataClient {
   ) {
     const setupStartTime = moment().utc().toISOString();
     const { logger, namespace, appClient, dataViewsService } = this.options;
-    const indexPatterns = await buildIndexPatterns(namespace, appClient, dataViewsService);
+    try {
```

Comment:
this is the fix, move the `try` higher :D

Reply:
uuugh, this is why i dont like try/catch blocks! 
I remember fixing an issue a couple weeks back caused by the exact same thing